webapp_frontend/src/background/api/api.ts

```diff
@@ -1,38 +1,38 @@
-import Axios, { } from "axios";
+import Axios from "axios";
 import { constants } from "../constants";
 
 export const hostname: string = constants.url.API_URL;
 
-export const userPath: string = '/v1/users';
+export const userPath: string = "/v1/users";
 
-export const filesystemPath: string = '/v1/filesystem';
+export const filesystemPath: string = "/v1/filesystem";
 
 enum DataIntegrity {
-    STABLE = "systemHealth/traffic_green.png",
-    POSSIBLE_RISK = "systemHealth/traffic_yellow.png",
-    UNSTABLE = "systemHealth/traffic_red.png"
+  STABLE = "systemHealth/traffic_green.png",
+  POSSIBLE_RISK = "systemHealth/traffic_yellow.png",
+  UNSTABLE = "systemHealth/traffic_red.png",
```

Comment:
Did i already mention the fact, that this repo is lacking some config on how to format.